src/phantom.js

```diff
@@ -75,9 +76,15 @@ export default class Phantom {
         this.process.on('error', error => {
             logger.error(`Could not spawn [${phantomjs.path}] executable. Please make sure phantomjs is installed correctly.`);
             logger.error(error);
+            this.kill('Process got an error');
             process.exit(1);
         });
 
+        this.process.stdin.on('error', (e) => {
+            logger.debug(`Child process received error ${e}, sending kill signal`);
+            this.kill('Error reading from stdin');
```

Comment:
Is this really what you want here? If any stdin error is outputted then the process just exits. That doesn't sound right. What if PhantomJS does prints a warning or error but everything else still works.